src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -91,6 +92,18 @@ object GardenVisitorFeatures {
         "gardenexperience",
         " §8\\+§2(?<amount>.*) §7Garden Experience"
     )
+    public val farmingXpPattern by patternGroup.pattern(
+        "farmingxp",
+        " §8\\+§3(?<amount>.*) §7Farming XP"
+    )
+    private val rewardsTextPattern by patternGroup.pattern(
+        "rewardstext",
+        "^ {2}§a§lREWARDS"
+    )
+    public val visitorRewardPattern by patternGroup.pattern(
```

Comment:
Please use regex tests, especially for complex regexes like this. When you have optional parameters, please use multiple test strings